guidelines/style-guidelines.md

```diff
@@ -0,0 +1,162 @@
+
+#### Colon and semi-colons
+
+We avoid these. When possible we’ll write 2 sentences instead, or separate the clauses using a dash (with a space on either side).
+
+#### Apostrophes
+
+We generally don’t add an extra 's' after nouns or names ending in 's'.
+For example: "The business’ work" – not  "the business’s work."
+
+We don’t use an apostrophe for dates, numbers, or plurals of abbreviations.
+For example: 1960s.
+
+#### Bold
+
+We rarely use bold – using too much will make it difficult for users to know which parts of your content they need to pay the most attention to. To emphasise words or phrases, we:
+* put key information at the start of sentences
+* use headings
+
+#### Capitals
+
+We only use capitals for proper nouns, such as:
+* names of people, places and things, including buildings and brands
+* titles, such as: of books, albums, recordings
+
+'Types' of places are not capitalized, unless referring to a specific place.
+For example: "I like the library" - and "I like my local Library"
+
+Generally, terms are not proper nouns, so should not be capitalised. Technical terms are not proper nouns. But if a word or term is branded as a distinct thing, treat it as a proper noun.
+
+#### Headings
+
+We use sentence case so only the first letter is upper case. We never link headings.
+
+#### Quotation marks
+
+We use double quotation marks for:
+* exact quotations
+* direct speech.
+
+We use single quotation marks for:
+* technical terms (the first time it is used)
+* classification descriptors
+* titles of documents or publications
+
+
+## Links
+
+We:
+* link to relevant content on our website before we link to external websites 
+* never link headings
+* write descriptive links that tell you what you’ll find when you follow them - not 'click here'
+* don't set links to open in a new tab or window, with exceptions to be decided case by case.
+
+
+## Lists
+
+We use bulleted lists (coded as unordered lists) to list items or points, and numbered lists (coded as ordered lists) for processes where the order of steps is important.
+
+We try to:
+* keep our lists short (2–7 items)
+* only use 1 level of nesting.
+
+We use 2 types of bulleted lists – single-sentence lists and multi-sentence lists.
+
+When we’re writing a single-sentence list, we:
+* start with a stem sentence that all the points have in common
+* start each point in lower case, and only use a full stop on the last point
+* don't use 'and' or 'or' on the second-to-last point
+* check that each point makes a full sentence when read with the stem.
+
+Multi-sentence lists are introduced by a complete sentence.
+* Each point in the list is also a complete sentence.
+* Each point can be 1–3 sentences long.
+* Each point begins with a capital letter and ends with a full stop.
+
+
+## Numbers
+
+In general:
+* we use numerals instead of words when we write numbers – this helps you scan our content
```

Comment:
Even if it's "you have two options" or "you can pick up to three"?

Reply:
I was very curious about this too! I always learned to write out the number in words up until... 8? or 10?

I do like the point here about helping to scan content. Makes it very easy to pick out numbers in a paragraph if you're looking for them. But happy to replace with 'use words up until x, then use numbers', if you think that's better.

Reply:
I think "use words from 0-9" and "use digits for 10 and up" seems reasonable.  I learned something similar (though I don't remember the exact cutoff).  One situation where I think digits look odd (especially for small numbers) is at the beginning of a sentence: "Two options are available." vs. "2 options are available."